walkthroughs/howto-tls-ecs-crossvpc/acm-cert.sh

```diff
@@ -0,0 +1,16 @@
+# Create the CA Cert
```

Comment:
This file does not appear to be necessary either?

Reply:
Removed the file